52.md

```diff
@@ -22,14 +22,14 @@ This kind of calendar event starts on a date and ends before a different date in
 
 The format uses a parameterized replaceable event kind `31922`.
 
-The `.content` of these events is optional and should be a detailed description of the calendar event.
+The `.content` of these events should be a detailed description of the calendar event. It is required but can be an empty string.
```

Comment:
I believe `content` is always expected on any kind. Please correct me if I'm wrong.